bot/src/main.rs

```diff
@@ -1416,6 +1416,17 @@ fn classify(
                     0
                 });
 
+            // if the commission was below max_commission last epoch, and is above max_commission this epoch
+            let commission_increased_above_max = commission_at_end_of_epoch > config.max_commission
+                && previous_classification
```

Comment:
1) for your comment on line 1419, when you say "this epoch" are you referring to the epoch boundary? Since we care about when commission is higher than max_commission at the epoch boundary (which is technically last epoch, not this epoch...)
2) what is the logical expression on previous_classification checking for?

Reply:
I just updated the comment to be more explicit. Commission is always the commission at the boundary between two epochs.

The expression on line 1424 is checking whether the commission at the end of the last epoch was below max_commission.